.github/release-drafter.yml

```diff
@@ -20,6 +20,7 @@ categories:
       - 'deprecated'  # fef2c0
 exclude-labels:
   - 'skip-changelog'
+  - 'bot:chronographer:skip'
```

Comment:
_(not really important but I'd prefer the entries to be sorted alphabetically when possible)_